pkg/ingester/ingester.go

```diff
@@ -372,6 +372,13 @@ func (i *Ingester) Push(ctx context.Context, req *client.WriteRequest) (*client.
 		}
 	}
 
+	if len(req.Metadata) > 0 {
```

Comment:
This is only here temporary, I'd like to some feedback on what I've done before I open a PR with the rest of the change.